<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>70</height>
       </size>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>62</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>物品名</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>62</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>规格</string>
       </property>
      </widget>
      <widget class="QPushButton" name="serachButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>4</y>
         <width>114</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>查找</string>
       </property>
      </widget>
      <widget class="QComboBox" name="namesComboBox">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>5</y>
         <width>111</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="etalonComboBox">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>34</y>
         <width>111</width>
         <height>26</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="addButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>32</y>
         <width>114</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>新增</string>
       </property>
      </widget>
      <widget class="QPushButton" name="delButton">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>4</y>
         <width>114</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
      <widget class="QPushButton" name="commitButton">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>32</y>
         <width>114</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>提交</string>
       </property>
      </widget>
      <widget class="QPushButton" name="commitButton_2">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>20</y>
         <width>114</width>
         <height>32</height>
        </rect>
       </property>
       <property name="text">
        <string>添加物品种类</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>日期</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>凭证单号</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>摘要</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>收入数量</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>长期借</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>流动借</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>损耗</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>遗失</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>库存(常借)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>库存(流借)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>库存(新)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>库存(旧)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>库存合计</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>合计</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
